packages/zowe-explorer-api/src/tree/IZoweTree.ts

```diff
@@ -134,6 +134,11 @@ export interface IZoweTree<T> extends vscode.TreeDataProvider<T> {
      * @param favorite Node to refresh
      */
     refreshElement(node: IZoweTreeNode): void;
+    /**
+     * Signals that node data has changed in the tree view
+     * @param element to pass to event listener callback
+     */
+    nodeDataChanged(node: IZoweTreeNode): void;
```

Comment:
Should this be added to the changelog?
As well as any required methods introduced by the extension of `vscode.TreeDragAndDropController` (if any)

Reply:
Not to sound like a broken record, but I'm sorry for these lack of changelog entries. Fixed in cae206c